WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListActivity.java

```diff
@@ -67,8 +75,10 @@ public void onClick(View view) {
                     long feedId = getIntent().getLongExtra(ReaderConstants.ARG_FEED_ID, 0);
                     if (feedId != 0) {
                         showListFragmentForFeed(feedId);
+                        mSiteId = feedId;
```

Comment:
We also need to define `mSiteId` when `savedInstanceState` is not `null`, like after configuration change.

Reply:
Yep!  I added variable restoration in 124e030.